Lib/_pyrepl/windows_console.py

```diff
@@ -111,6 +111,7 @@ def __init__(self, err: int | None, descr: str | None = None) -> None:
 MOVE_UP = "\x1b[{}A"
 MOVE_DOWN = "\x1b[{}B"
 CLEAR = "\x1b[H\x1b[J"
+CLEAR_ALL= "\x1b[3J\x1b[2J\x1b[H"
```

Comment:
```suggestion
CLEAR_ALL= "\x1b[H\x1b[3J\x1b[J"
```
Using this works for me in the new virtual console on Windows. Also more symmetric to the `CLEAR` above.